src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -252,6 +258,74 @@ export function fetchFacilityDetails(facilityId) {
   };
 }
 
+export function openFacilityPageV2(page, uiSchema, schema) {
```

Comment:
This action borrows heavily from the existing `openFacilityPage()` and `updateFacilityPageData()` actions.  Since we are skipping the step of parent selection, it fetches all parent facilities, then child facilities for each parent.